api/src/main/java/graphql/nadel/hooks/CreateServiceContextParams.java

```diff
@@ -1,79 +1,32 @@
 package graphql.nadel.hooks;
 
-import graphql.Internal;
 import graphql.PublicApi;
-import graphql.execution.ExecutionContext;
-import graphql.execution.ExecutionId;
-import graphql.execution.ExecutionStepInfo;
 import graphql.nadel.Service;
-import graphql.nadel.engine.NadelContext;
-import graphql.schema.GraphQLSchema;
 
 @PublicApi
 public class CreateServiceContextParams {
     private final Service service;
-    private final ExecutionStepInfo executionStepInfo;
-    private final ExecutionId executionId;
-    private final GraphQLSchema schema;
-    private final NadelContext context;
```

Comment:
Not used, the new engine only supplies service. I've also checked with AGG and it seems fine.